app/js/globals.js

```diff
@@ -4,6 +4,7 @@ require.config({
   baseUrl: '/',
   paths: {
     'gaia-dialog': 'components/gaia-dialog/gaia-dialog',
-    'gaia-component': 'components/gaia-component/gaia-component'
+    'gaia-component': 'components/gaia-component/gaia-component',
+    'gaia-icons': 'components/gaia-icons/gaia-icons'
```

Comment:
I've tested this on my main dogfooding device, and as a web page. Seems to work ok.